api/egl.xml

```diff
@@ -1064,8 +1068,8 @@
 
 <!-- Reservable for future use. To generate a new range, allocate multiples
      of 16 starting at the lowest available point in this block. -->
-    <enums namespace="EGL" start="0x34E0" end="0x3FFF" vendor="KHR" comment="Reserved for future use">
-            <unused start="0x34E0" end="0x3FFF"/>
+    <enums namespace="EGL" start="0x34F0" end="0x3FFF" vendor="KHR" comment="Reserved for future use">
+            <unused start="0x34F0" end="0x3FFF"/>
```

Comment:
<enums namespace="EGL" start="0x350F0" end="0x3FFF" vendor="KHR" comment="Reserved for future use">
            <unused start="0x3500" end="0x3FFF"/>

Reply:
The reserve for future use now starts at 0x3500